docs(readme): add instructions for EL, notes for extras (#5341)
terrapkg/packages: README.md

## Installation

### Fedora

```bash

> sudo dnf install --nogpgcheck --repofrompath 'terra,https://repos.fyralabs.com/terra$releasever' terra-release

Reviewer: add instructions to install EPEL

Reviewer 2: Instead of this could we make the Terra repo package dep on this on EL?

Author: let's do both

sudo rpm-ostree install terra-release
```

> In addition, install `terra-release-extra` for more repositories (e.g. `terra-nvidia`, `terra-mesa`, `terra-extras`).

Reviewer 2: Maybe "in addition, you can"? The current wording could possibly be read as that you should/need to install these.

Author: tru

> In addition, you can install `terra-release-extra` for more repositories (e.g. `terra-nvidia`, `terra-mesa`, `terra-extras`).

Reviewer: ```suggestion
Optionally, you can install `terra-release-extra` to use the Extras repository. This also installs the Nvidia, and Mesa streams but does not enable them.
```

fact check required

### Enterprise Linux (EL)

> Only EL10 is supported. Not all packages available in Terra are available in Terra EL at this time.

Reviewer 3: Add that EL deps on EPEL?

Author: wdym

> Only EL10 or above are supported, and not all packages available in Terra are available in Terra EL.

Reviewer: ```suggestion
Only EL10 is supported. Not all packages available in Terra are available in Terra EL at this time.
```
